manifests/a/AudibleInc/AudibleSync/1.8.8/AudibleInc.AudibleSync.installer.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: AudibleInc.AudibleSync
+PackageVersion: 1.8.8
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://windows-lite-update.s3.amazonaws.com/windows/v1.8.8/AudibleSync+Setup+1.8.8.exe?pf_rd_p=414475fc-cc54-4d14-838c-b7c61b8f72c7
+  InstallerSha256: BA1228FB9E37D142700EF63A6CBB838B97E3745057E39B4D6B9086364D3167E9

```

```suggestion
  InstallerSha256: BA1228FB9E37D142700EF63A6CBB838B97E3745057E39B4D6B9086364D3167E9
  InstallerSwitches: 
      Silent: " "
      SilentWithProgress: " "
```
To stop the warning for `winget validate`.